Dockerfile

```diff
@@ -36,6 +36,13 @@ LABEL io.k8s.display-name="internal-services"
 LABEL io.openshift.tags="internal-services"
 LABEL summary="Konflux Internal Services"
 LABEL com.redhat.component="internal-services"
+LABEL org.opencontainers.image.created="2025-11-24"
+LABEL vendor="Red Hat Inc"
+LABEL distribution-scope="public"
+LABEL release="0"
+LABEL url="github.com/konflux-ci/internal-services"
+LABEL version="0"
```

Comment:
Is there a plan to update this version and release labels or we just keep them at 0?